pathod/language/http.py

```diff
@@ -191,7 +191,7 @@ def resolve(self, settings, msg=None):
         if self.ws:
             if not settings.websocket_key:
                 raise exceptions.RenderError(
-                    "No websocket key - have we seen a client handshake?"
+                    b"No websocket key - have we seen a client handshake?"
```

Comment:
Yeah, you're correct. It doesn't have to be. I was just blindly converting literals to bytes. Will fix this!